chapters/annotations.tex

```diff
@@ -1340,7 +1372,7 @@ \section{Graphical User Interface}\label{annotations-for-the-graphical-user-inte
 
 A simple type or component of a simple type may have:
 \begin{lstlisting}[language=modelica]
-annotation(absoluteValue = false);
+/*literal*/ constant Boolean absoluteValue;
```

Comment:
```suggestion
/*literal*/ constant Boolean absoluteValue=true;
```
Will add a bit more a few lines below. Having `true` as default makes sense, as that means that unit offset is used for a variable just having `unit="K"`.